Pocket.xcodeproj/project.pbxproj

```diff
@@ -1853,17 +1845,20 @@
 			repositoryURL = "https://github.com/braze-inc/braze-swift-sdk.git";
 			requirement = {
 				kind = exactVersion;
-				version = 5.13.0;
+				version = 5.14.0;
+			};
+		};
+		6547A0FD29F17AAF00F6FC6B /* XCRemoteSwiftPackageReference "Sails" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/Pocket/Sails.git";
+			requirement = {
+				branch = master;
```

Comment:
How can we hard-version Sails?

Reply:
this thing hasnt been updated nor do i think we plan to right now